### PR TITLE
hdt 1.3.3 (new formula)

### PR DESCRIPTION
**Pull request by ptorrestr**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/hdt.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+class Hdt < Formula
+  desc "Header Dictionary Triples (HDT) is a compression format for RDF data"
+  homepage "https://github.com/rdfhdt/hdt-cpp"
+  url "http://srvgal80.deri.ie/~pabtor/hdt/hdt-1.3.3.tar.gz"
+  sha256 "ac624802909ce04cbc046f00173496fb44768b45157d284d258d394477ae23b9"
+
+  depends_on "serd"
+  depends_on "zlib"
+
```
Comment (SMillerDev) [inline Formula/hdt.rb]: `depends_on "pkg-config" => :build`

File: Formula/hdt.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
+class Hdt < Formula
+  desc "Header Dictionary Triples (HDT) is a compression format for RDF data"
+  homepage "https://github.com/rdfhdt/hdt-cpp"
+  url "http://srvgal80.deri.ie/~pabtor/hdt/hdt-1.3.3.tar.gz"
+  sha256 "ac624802909ce04cbc046f00173496fb44768b45157d284d258d394477ae23b9"
+
+  depends_on "pkg-config" => :build
+  depends_on "serd"
+  uses_from_macos "zlib"
+
```
Comment (SMillerDev) [inline Formula/hdt.rb]:
```suggestion
  depends_on "serd"

  uses_from_macos "zlib"

```

**Comment by SMillerDev** [comment]
Could you squash your commits into one that says `<name> <version> (new formula)`?

**Comment by chenrui333** [comment]
@ptorrestr Thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**